packages/wallet/src/persistence/types.ts

```diff
@@ -81,6 +82,8 @@ export interface WalletStores extends Destroyable {
   unspendableUtxo: CollectionStore<Cardano.Utxo>;
   transactions: OrderedCollectionStore<Cardano.TxAlonzo>;
   inFlightTransactions: DocumentStore<Cardano.NewTxAlonzo[]>;
+  // REVIEW: i'm not sure what type of store would be suitable here
+  confirmedTransactions: DocumentStore<ConfirmedNewTxAlonzoWithSlot[]>;
```

Comment:
Would be great to come up with a better name for this store, to indicate that those are the transactions of not 100% confidence (within rollbackable range)

Reply:
A few options:
`stabilityWindowTransactions`
`revertableTransactions`
`pendingChainConfirmationTransactions`

Reply:
I like `stabilityWindowTransactions`

Reply:
Updated in [0286cb7](https://github.com/input-output-hk/cardano-js-sdk/pull/351/commits/0286cb7b5c3844765304acc03f3163345df65fa2)